e2e/e2eutil/input/disconnect-node.nomad

```diff
@@ -13,6 +13,11 @@ job "disconnect-node" {
 
   group "group" {
 
+    reschedule {
```

Comment:
@tgross  I don't know that this is required. I just noticed during testing, that a new alloc would immediately get scheduled for the disconnected node which seemed like another thing that could potentially go wrong or might result in flakiness. I can remove if you don't think it's necessary or if you had excluded this on purpose.

Reply:
Oh interesting. No that wasn't intentional. This seems like a good change.

Reply:
I'll do that now and merge